Makefile

```diff
@@ -180,6 +181,16 @@ commons-all:
 	cp obiba-shiro-crowd/target/obiba-shiro-crowd-${commons_version}.jar ${opal_project}/opal-server/target/opal-server-${version}/lib
 
 #
+# Compile and install jennite
+#
+jennite:
+	cd ${jennite_project} && \
+	${mvn_exec} clean install && \
+	mkdir -p ${opal_home}/plugins/tmp && \
+	rm -rf ${opal_home}/plugins/* && \
+	unzip ${jennite_project}/jennite-vcf-store/target/*-dist.zip -d ${opal_home}/plugins/
```

Comment:
it didn't works ...
jennite:
	cd ${jennite_project} && \
	${mvn_exec} clean install && \
	cp ${jennite_project}/jennite-vcf-store/target/*-dist.zip ${opal_home}/plugins/
the plugin is not loaded